src/panels/config/automation/manual-automation-editor.ts

```diff
@@ -572,6 +574,24 @@ export class HaManualAutomationEditor extends LitElement {
     });
   }
 
+  private _getCollapsableElements() {
+    return this.shadowRoot!.querySelectorAll<
```

Comment:
Use a decorator here `@queryAll`

Reply:
I thought it's maybe more efficient to just query when needed?

Reply:
The decorator also only query when needed, it does the same, just a different (shorter) syntax.